docs/guides/databases/mariadb/backup-mariadb-mysql-to-object-storage-with-restic/index.md

```diff
@@ -131,29 +147,29 @@ ID        Time                 Host        Tags        Paths
 1 snapshots
 {{< /output >}}
 
-### Run the backup script automatically
+## Set Up Automated Database Backups
 
 Linux has several ways of running a job on a schedule basis. Pick one of the methods below to configure the backup script to run periodically.
 
 {{< note >}}
 Consider your databases' usage, how much data you could potentially lose, and the storage space required when choosing how often to run your script.
 {{< /note >}}
 
-### 1. Cron
```

Comment:
The table of contents in the new UI has header-numbering (and -lettering) displayed before each title by default, so the numbering here ended up being redundant